pkg/datapath/iptables/ipset/ipset.go

```diff
@@ -52,11 +52,12 @@ type Initializer interface {
 type initializer struct {
 	once sync.Once
 	wg   *lock.StoppableWaitGroup
+	done lock.DoneFunc
 }
 
 func (i *initializer) InitDone() {
 	// further calls to InitDone will be a no-op
-	i.once.Do(i.wg.Done)
+	i.once.Do(i.done)
```

Comment:
Nit. the `done` function is now already wrapped by `once`, and the extra one here could be dropped.